lib/function/arithmetic/log10.js

```diff
@@ -31,7 +31,9 @@ function factory (type, config, load, typed) {
   var log10 = typed('log10', {
     'number': function (x) {
       if (x >= 0 || config.predictable) {
-        return Math.log(x) / Math.LN10;
+        return (Math.log10)
```

Comment:
This check needs to be done only once on load. For optimization we could move this into a separate function, like:

```js
var _log10number = Math.log10
  ? Math.log10 
  : function (x) {
    return Math.log(x) / Math.LN10;
  };
```

Reply:
JS engines do not optimize these functions yet. They are implemented as plain JavaScript, which is the reason for equivalent performance.